cpp/src/arrow/engine/substrait/extension_set.cc

```diff
@@ -592,6 +654,10 @@ struct ExtensionIdRegistryImpl : ExtensionIdRegistry {
 
   Result<SubstraitAggregateToArrow> GetSubstraitAggregateToArrow(
       Id substrait_function_id) const override {
+
+    if (substrait_function_id.uri == kArrowSimpleExtensionFunctionsUri) {
```

Comment:
TODO: Revert this change after https://github.com/apache/arrow/pull/35608 is merged

Reply:
Done